<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="d_sort2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="l_data_32">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Sortuj pomiary według:</string>
   </property>
  </widget>
  <widget class="QLabel" name="widget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>165</width>
     <height>168</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>541</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_data_22">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Wybierz datę:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_data2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>2019-09-02</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_all2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>POKAŻ WSZYSTKIE POMIARY TEMPERATURY</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>312</width>
     <height>183</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b_show2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>POKAŻ DANE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
